### PR TITLE
fix tests for des-895-900

### PR DESCRIPTION
**Pull request by umahaea**
- fix tests in progress bar
- fix tests in buttons - disabled
- fix tests in alerts

### CODE REVIEW

**Comment by StommePoes** [comment]
Seems okay to me. I added the display: flex stuff to alerts, then eyeballed that they looked like the designs and the current live demo on Chrome. I checked that they looked ok on FF and IE Windows 7.

I could not eyeball on a mobile and maybe that should be done before releasing?

**Comment by umahaea** [comment]
yes mallory, flex is verified on mobile as well.

File: src/test/java/elementsSDKTests/functionalTests/AlertsTest.java [hunk 1]
```diff
@@ -422,17 +436,23 @@ private void alertTextPropertiesMobileTest(String alertClass, String alertType,
...
         paddingRight = commonUtils.getCSSValue(compAlertsPgObj.alertContent, "padding-right", "mobile");
 
         isMarginLeft = commonUtils.assertValue(marginLeft, expMarginLeft, "Compounds -> Margin left of alert " + alertClass + alertType + " is not as per spec");
+        isMarginTop = commonUtils.assertCSSProperties("margin-top", marginTop, expMarginTop);
+        if (!isMarginTop) {
+            log.info("Compounds -> Margin top of alert " + alertClass + alertType + " is not as per spec, actual: " + marginTop);
+        }
+
         isPaddingTop = commonUtils.assertValue(paddingTop, "4px", "Compounds -> Padding top of alert " + alertClass + alertType + " is not as per spec");
         isPaddingRight = commonUtils.assertCSSProperties("padding-right", paddingRight, new String[]{"8px", "12px"});
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/AlertsTest.java]: assertion for marginTop missing here too

**Comment by udhadpa** [comment]
Other than my comments. Rest looks good 👍 It was a super quick fix.

**Comment by briandunnpearson** [comment]
Approved!

**Comment by umahaea** [comment]
got many approvals. good to merge.

**Pull request merged**